code/modules/organs/subtypes/shadekin.dm

```diff
@@ -2,4 +2,5 @@
 	can_assist = FALSE
 
 	var/dark_energy = 100
-	var/max_dark_energy = 100
\ No newline at end of file
+	var/max_dark_energy = 100
+	var/dark_energy_infinite = FALSE
```

Comment:
Badass var name.

Reply:
infinity_unlimited_flame_of_incomprehensible_dark_beyond was a bit of a mouthful

Reply:
Could always acronym it!

Reply:
:joy_cat: